Gem/Resources/ar.lproj/Localizable.strings

```diff
@@ -190,4 +190,5 @@
 "wallets.watch" = "Watch";
 "transfer.minimum_amount" = "الحد الأدنى للمبلغ هو %@";
 "wallet_connect.no_active_connections" = "No active connections";
-"stake.minimum_amount" = "الحد الأدنى للمبلغ";
\ No newline at end of file
+"stake.minimum_amount" = "الحد الأدنى للمبلغ";
+"common.no_results_found" = "No Results found";
```

Comment:
can you rename `No Results found` => `No Results Found` to all cases?

Reply:
👌